introduction.md

```diff
@@ -0,0 +1 @@
+Hi there!  My name is Zack and I'm going to be a blackbelt.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.